app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :user, foreign_key: 'users_id'
```

Comment:
A like should belong to an `author` instead of a user.

Reply:
Hi @TheLuisAbarca , thanks for reviewing my job and for your kind feedback. The changes are implemented in commit 90ae214.